model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -349,7 +348,8 @@ def _get_process_output(self):
         result = ""
         try:
             result = tmp_output.decode("utf-8")
-        except:
+        except Exception:
+            # Ignore the result on decode failed
```

Comment:
Unnecessary comment?

Reply:
This comment is intended to resolve https://github.com/triton-inference-server/model_analyzer/security/code-scanning/7